coderd/database/models.go

```diff
@@ -1,6 +1,6 @@
 // Code generated by sqlc. DO NOT EDIT.
 // versions:
-//   sqlc v1.20.0
+//   sqlc v1.24.0
```

Comment:
You'll need to get your environment matching CI --- easiest is to use the Coder on Coder template, or Nix.